mne/io/eeglab/eeglab.py

```diff
@@ -258,6 +312,18 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
 
         last_samps = [eeg.pnts - 1]
         info = _get_info(eeg, montage, eog=eog)
```

Comment:
Can we do that in another issue, where we deal with Misc in general? This one is confusing enough for me already I must admit.